src/Microsoft.ComponentDetection.Detectors/nuget/NuGetProjectModelProjectCentricComponentDetector.cs

```diff
@@ -293,30 +140,15 @@ private void NavigateAndRegister(
             else
             {
                 visited.Add(dependency.Id);
-                this.NavigateAndRegister(target, explicitlyReferencedComponentIds, singleFileComponentRecorder, targetLibrary, libraryComponent.Component.Id, dotnetRuntimePackageNames, visited);
+                this.NavigateAndRegister(target, explicitlyReferencedComponentIds, singleFileComponentRecorder, targetLibrary, libraryComponent.Component.Id, frameworkPackages, visited);
             }
         }
-    }
 
-    private bool IsAFrameworkComponent(HashSet<string> frameworkComponents, string libraryName, IList<PackageDependency> dependencies = null)
-    {
-        var isAFrameworkComponent = frameworkComponents.Contains(libraryName);
-
-        if (isAFrameworkComponent)
-        {
-            this.frameworkComponentsThatWereOmmittedWithCount.AddOrUpdate(libraryName, 1, (name, existing) => existing + 1);
-
-            if (dependencies != null)
-            {
-                // Also track shallow children if this is a top level library so we have a rough count of how many things have been ommitted + root relationships
-                foreach (var item in dependencies)
-                {
-                    this.frameworkComponentsThatWereOmmittedWithCount.AddOrUpdate(item.Id, 1, (name, existing) => existing + 1);
-                }
-            }
-        }
+        // a placeholder item is an empty file that doesn't exist with name _._ meant to indicate an empty folder in a nuget package, but also used by NuGet when a package's assets are excluded.
+        bool IsAPlaceholderItem(LockFileItem item) => Path.GetFileName(item.Path).Equals("_._", StringComparison.OrdinalIgnoreCase);
```

Comment:
Any links to documentation about this `_._`, I have never seen this beofre.

Reply:
Here's an old discussion https://github.com/dotnet/aspnetcore/issues/744#issuecomment-123411563

SDK does the same: https://github.com/dotnet/sdk/blob/73bde768ff9058723021da4a9b28f93ff85f8952/src/Tasks/Microsoft.NET.Build.Tasks/NuGetUtils.NuGet.cs#L11

There used to be public docs on this, when creating a nuspec you'd use this file to create an empty directory, so that it could represent the package would install but wouldn't provide any assets.  It was important when representing a assembly that was inboxed in a later framework, or that targets weren't needed on a framework, etc.  NuGet reused it when implementation `ExcludeAssets` so that it would look the same as if a package included an empty folder.

Reply:
I updated the public docs to mention this: https://github.com/NuGet/docs.microsoft.com-nuget/pull/3348/files

I also changed this to use the public API: `PackagingCoreConstants.EmptyFolder`